filter_directories.py

```diff
@@ -13,7 +19,10 @@
     while directory:
         if os.path.exists( os.path.join(directory, '.shed.yml') ) or os.path.exists( os.path.join(directory, '.shed.yaml') ):
             if directory not in bl:
-                print(directory)
+                for filename in os.listdir(directory):
```

Comment:
I suspect this may break the automatic push to TTS if there is more than one tool in a directory.